### DISCUSSION TITLE
I verified my Student status, and i really want to use copilot!

### DISCUSSION THREAD

**Discussion by sigmaith** [community/community#132878] [GitHub Education]
### Hi everyone,

I verified my Student status, and i really want to use copilot!

**Comment by davevad93** [comment]
Hi @sigmaith ,

If you can't get access to copilot [here](https://education.github.com/pack/redeem/copilot-student), you can try to install the `Copilot` extension in your IDE e.g. `VS Code`, `Visual Studio`, `IntelliJ`, `Android Studio`, `R Studio` etcetera.

I'm gonna show you the steps for `VS Code`:

1. Install the Copilot extension in VS Code.
2. Log in with your current GitHub account.
3. Copy the 8-digit verification code to the clipboard and paste it into the `Device Activation` pop-up window.

Some users who got their `Student Developer Pack` benefits but couldn't access the free `Copilot` subscription solved it by doing this, so give it a try.

**Reply by amanaknows** [reply]
Don't listen to this clown, just go in the sidebar.

**Reply by sigmaith** [reply]
Uhm.. i just get qualified access to Copilot and update my Copilot plugin, and reboot my intellij. I solved it Thx everyone!!

**Reply by davevad93** [reply]
I'm glad you solved it @sigmaith , have a nice day!

**Reply by neurafusionai** [reply]
glad you solved it!
